packages/features/auth/lib/next-auth-options.ts

```diff
@@ -807,22 +807,6 @@ export const AUTH_OPTIONS: AuthOptions = {
               },
             });
 
-            // safely delete password from UserPassword table if it exists
-            try {
```

Comment:
Previously when user created an account and then logged in via google then his password field was deleted, now we no longer do that because we let users log in via google and email + password